_components/card/card.md

```diff
@@ -32,8 +32,6 @@ variants:
     description: In combination with `usa-card--flag`, sets the media element on the right. (Flag cards display media on the left by default.)
   - variant: "`.usa-card__media--inset`"
     description: Indents the media element so it doesn't extend to the edge of the card.
-  - variant: "`.usa-card__media--set-aspect`"
```

Comment:
This variant is no longer needed. Users can directly apply add-aspect utility classes without needing to use this set-aspect class.